src/sentry/api/endpoints/group_tagkey_values.py

```diff
@@ -74,5 +75,19 @@ def get(self, request, group, key):
             queryset=queryset,
             order_by=order_by,
             paginator_cls=paginator_cls,
-            on_results=lambda x: serialize(x, request.user, serializer_cls),
+            on_results=lambda results: serialize(
+                map(  # XXX: This is a pretty big abstraction leak
```

Comment:
This is gross but I didn't want to take the time to push this down into the tagstore backend right now.